sample/src/com/vungle/publisherSampleAndroid/MainActivity.java

```diff
@@ -35,12 +37,12 @@ public void onCreate(Bundle savedInstanceState) {
 		setContentView(R.layout.activity_main);
 
 		// get your App ID from the app's main page on the Vungle Dashboard after setting up your app
-		final String app_id = "Test_Android";
+		final String app_id = APP_ID;
```

Comment:
If we're keeping the string as a constant in the class scope instead, we don't need to duplicate the variable.  Let's just init with APP_ID instead of app_id